src/components/card-rate.tsx

```diff
@@ -0,0 +1,26 @@
+import Rate from './rate';
+
+const STARS_COUNT = 5;
+const STARS = Array.from({ length: STARS_COUNT });
+
+type CardRateProps = {
+  rating: number;
+  reviewCount : number;
+}
+
+function CardRate({rating, reviewCount}: CardRateProps): JSX.Element {
```

Comment:
```suggestion
function CardRate({rating, reviewCount}: CardRateProps)t {
```
Наверняка : JSX.Elemen тип избыточен здесь и во всех других файлах. TS. и так понимает что это компонент